data/README.md

```diff
@@ -0,0 +1,10 @@
+# How to get desired data
+## Data source
+In our case desired data (sounds of birds) comes from .......
```

Comment:
It's okay, but maybe for clarity it would be a good idea to describe it better. Maybe with GPT 😛

Reply:
In the future ofc, that's why it's not an issue in PR